control-plane/grpc/src/tracing.rs

```diff
@@ -239,7 +237,11 @@ fn update_span_from_response<T>(context: opentelemetry::Context, response: &Resp
         }
         None => tonic::Code::Ok,
     };
+    const RPC_GRPC_STATUS_CODE: &str = "rpc.grpc.status_code";
```

Comment:
Why do we need to define it now? Is it removed from the previous imported path?

Reply:
Yes it was removed, it's now under some unsafe feature export, so I rather just hardcode it